tensornetwork/ncon_interface.py

```diff
@@ -17,19 +17,23 @@
 from __future__ import division
 from __future__ import print_function
 import warnings
-from typing import Any, Sequence, List, Optional, Union, Text, Tuple, Dict
+from typing import Any, Sequence, List, Optional, Union, Text, Tuple, Dict, Set
 from tensornetwork import network
 from tensornetwork import network_components
-
+# pylint: disable=useless-import-alias
```

Comment:
If they're useless why did you add them?

Reply:
(Just do `from tensornetwork import config`)

Reply:
fixed